homework16/js/main.js

```diff
@@ -0,0 +1,211 @@
+const API_KEY = '91d38c9e';
+const searchInput = document.querySelector('.search-films__input');
+const searchButton = document.querySelector('.search-films__button');
+let pageNum;
+
+function notFoundError(data){
+    const p = document.createElement('div');
+    p.innerText = `${data.Error}`;
+    p.className = 'search-error';
```

Comment:
Mostly we use `classList` to manage classes of element
Because we can add or remove a class without affecting any others the element may have. 
But if we assign `className`, it will wipe out any existing classes while adding the new one.